lapis2/src/main/kotlin/org/genspectrum/lapis/request/AminoAcidMutation.kt

```diff
@@ -26,15 +26,15 @@ data class AminoAcidMutation(val gene: String, val position: Int, val symbol: St
             return AminoAcidMutation(
                 gene,
                 position,
-                matchGroups["symbolTo"]?.value,
+                matchGroups["symbolTo"]?.value?.toString()?.uppercase(),
```

Comment:
IntelliJ claims that the `toString` is redundant
```suggestion
                matchGroups["symbolTo"]?.value?.uppercase(),
```

Reply:
Same for NucleotideMutation